docroot/docs/index.php

```diff
@@ -464,7 +464,7 @@ function renderEndpoint($endpoint, $xml=null) {
                 </div>
             </div>
 <?php
-        if ( count($endpoint->parameters->parameter) > 0 ) {
+        if ( count($endpoint->parameters->parameter > 0) ) {
```

Comment:
Is this a bug?

Reply:
No idea if this is a bug or not... I don't know about this function.

Reply:
We'll leave it for now. The same change is made lower in this file, so it seems intentional.
Though I'm not sure what this page is supposed to be. It's not working right now: https://api.helioviewer.org/docs 
The real docs page is up though, https://api.helioviewer.org/docs/v2